_posts/2022-01-10-reassign-partitions.md

```diff
@@ -42,17 +34,38 @@ Some of these are listed here:
 1. You can reassign the partition between the brokers anytime. For example, it can be used at times when you want to scale down the number of brokers in your cluster.
    You can assign partitions from the broker to be scaled down to other brokers which will handle these partitions now.
 
-2. With the help of this tool, you can increase the no. of partitions / replicas which can help with increasing the throughput of the topic.
+2. With the help of this tool, you can increase the number of partitions / replicas which can help with increasing the throughput of the topic.
+
+## Partition Reassignment Throttle
+Partition reassignment tool reassigns the partition between the brokers and sometimes this process could be slow due to transfer of large data.
+To avoid any sort of impact on the client we can throttle the reassignment process.
+Kafka lets us apply throttle to the replication traffic which sets an upper bound on the bandwidth used to move partition between the brokers.
+For example, --throttle 5000000 sets a maximum threshold for moving partitions of 50 MBps.
+
+Throttling might cause the reassignment to take longer to complete.
+
+1. If the throttle is too low, the newly assigned brokers will not be able to keep up with records being published and the reassignment will never complete.
+
+2. If the throttle is too high, clients will be impacted.
```

Comment:
More than just the clients, the overall health of the cluster may be impacted, for example if replication of other topics can't keep up!